ribs/archives/_cvt_archive.py

```diff
@@ -157,6 +159,8 @@ def __init__(self,
         self._centroid_kd_tree = None
         self._ckdtree_kwargs = ({} if ckdtree_kwargs is None else
                                 ckdtree_kwargs.copy())
+        self.chunk_size = chunk_size
+        self.chunk_distances = chunk_distances
```

Comment:
Remove this

Reply:
I removed the chunk_distances variable. self._chunk_size assignment is still necessary though